Code/BuildSystem/CMake/BuildFilters/PrecompiledTools.BuildFilter

```diff
@@ -27,4 +27,8 @@ set_property(GLOBAL PROPERTY EZ_BUILD_FILTER_PrecompiledTools
 	Utilities
 	ads
 	tinyexr
+    DependencyAnalysis
+    Lua
+    Duktape
+    Imgui
```

Comment:
Why do we have these dependencies in the precompiled tools? Was that always the case?